### PR TITLE
AF-154 add doc

### PR DESCRIPTION
**Pull request by sandlerr**
:koala:

/cc @zendesk/quokka @chucknado 

### References
 - Jira link: https://zendesk.atlassian.net/browse/AF-154

### CODE REVIEW

**Comment by pmgrove** [comment]
@sandlerr are we able to see an example of how the settings within the manifest are shown in the settings.yml file?

**Comment by danielbreves** [comment]
Why do we need the -c option? Why not always use that if present?

**Comment by sandlerr** [comment]
so you don't have to delete / edit the file when you want to run a one-off config? I don't feel strongly about it, happy to change if it you like.

**Comment by danielbreves** [comment]
I don't feel strongly either, it's only a couple of characters :)

**Comment by danielbreves** [comment]
:+1:

File: doc/tools.md [hunk 1]
```diff
@@ -54,6 +54,39 @@ Follow these steps to run an app locally:
 
 To stop the server, switch to your command-line interface and press Control+C.
 
+#### App Settings
+
+In testing some apps it is necessary to specify the settings for the app. `zat server` will ask interactively for the values of all settings specified in the app's `manifest.json`. However, `zat server -c [$CONFIG_FILE]` allows specifying the settings using a JSON or YAML file where the keys are the same as in the manifest. The default filename is `settings.yml`.
```
Comment (chucknado) [inline doc/tools.md]: Suggested edit for the first paragraph: 

> When testing your app, you might need to specify some [app settings](./manifest#app-settings). If you started the local server with  `zat server`, the app will ask interactively for the values of all the settings specified in the app's `manifest.json` file. However, you might prefer to specify the settings in a JSON or YAML file. 

> 1. Create a JSON or YAML file with your settings. The keys in the file should be the same as in your manifest file.
> 2. Start the server with `zat server -c [$CONFIG_FILE]`, where $CONFIG_FILE is the name of your JSON or YAML file. The default filename is `settings.yml`. You don't need to specify it if you use it.

File: doc/tools.md [hunk 2]
```diff
@@ -54,6 +54,39 @@ Follow these steps to run an app locally:
...
+
+As a concrete example if `manifest.json` contains
```
Comment (chucknado) [inline doc/tools.md]: Suggested edit:

> ##### Example
> If `manifest.json` contains the following settings:

File: doc/tools.md [hunk 3]
```diff
@@ -54,6 +54,39 @@ Follow these steps to run an app locally:
...
+```json
+...
+"parameters": [
+  {
+    "name": "mySetting"
+  },
+...
+```
+
+There may be settings files like
```
Comment (chucknado) [inline doc/tools.md]: Original:

> There may be settings files like

Suggested edit:

> Then you can specify the settings in a file as follows:

File: doc/tools.md [hunk 4]
```diff
@@ -54,6 +54,39 @@ Follow these steps to run an app locally:
...
+```
+
+or
+
+```yaml
+./settings.yml
+mySetting: test value
+```
+
+These may be used by either `zat server -c settings.json` or `zat server -c`.
```
Comment (chucknado) [inline doc/tools.md]: Original: 

> These may be used by either `zat server -c settings.json` or `zat server -c`.

Suggested edit:

> With the first file, you'd start the server with `zat server -c settings.json`. With the second file, you'd start it with `zat server -c`.

File: doc/tools.md (hunk 1 above)
Comment (sandlerr) [inline doc/tools.md]: instead of `the app will ask interactively` I put `ZAT will ask interactively`

**Pull request merged**